src/main/java/controllers/UserProfileController.java

```diff
@@ -226,4 +226,53 @@ private void runCustomizeBio() {
         String newBio = this.inOut.getInput("Enter what you'd like your new bio to be: ");
         this.runChangeBioDisplay(this.loginManager.getCurrUser(), newBio);
     }
+
+    /**
+     * Generate a selection of displayed posts' titles for the user to select.
+     *
+     * @param userPosts A hashmap of Posts representing a user's posts.
+     * @return a String representing all the posts of a user.
+     */
+    public String generateDisplayedPosts(HashMap<Integer, Post> userPosts) {
+        StringBuilder postsString = new StringBuilder("Enter a post number to view the post: ");
```

Comment:
Are we showing the user a list of posts to select from? We can't expect the user to memorize all the post numbers.

Reply:
the post titles is given with the post numbers in the prompt